auth/hello-world/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
```

Comment:
Required for the existing build test - else build run will run a typescript check across node modules